src/main/java/com/psk/pyragas/ToolRent/utils/interceptors/MethodLogger.java

```diff
@@ -14,24 +14,24 @@
 public class MethodLogger implements Serializable {
 
     @Inject
-    private FileIO fileWriter;
-
-    private final String logFileLocation = "methodLogs.txt";
+    private LoggerToFile logger;
 
     @AroundInvoke
     public Object logMethodInfo(InvocationContext context) throws Exception {
         String logMessage = "";
         logMessage += "--METHOD LOGGER START--" + "\n";
         logMessage += "Time before method call: " + new Timestamp(System.currentTimeMillis()) + "\n";
-        fileWriter.writeToTextFile(logFileLocation, logMessage);
+        logMessage += "Entering method: " + context.getMethod().getName() + "\n";
```

Comment:
I deleted these, because context doesn't tell us anything about the previous method. But now I realize it's good to have this. If there is an exception after the method is executed, then we'll see these.